plugins/callback/opentelemetry.py

```diff
@@ -0,0 +1,374 @@
+# (C) 2021, Victor Martinez <VictorMartinezRubio@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author: Victor Martinez (@v1v)  <VictorMartinezRubio@gmail.com>
+    name: opentelemetry
+    type: notification
+    short_description: create distributed traces with OpenTelemetry.
+    version_added: historical
```

Comment:
```suggestion
    version_added: 3.5.0
```
This plugin wasn't in ansible 0.x or 1.x before version_added was tracked, so `historical` is wrong.